pulsar-client-go/examples/consumer-listener/consumer-listener.go

```diff
@@ -22,7 +22,7 @@ package main
 import (
 	"fmt"
 	"github.com/apache/pulsar/pulsar-client-go/pulsar"
-	"log"
+	log "github.com/pulsar/pulsar-client-go/logutil"
```

Comment:
should this be instead `"github.com/apache/pulsar/pulsar-client-go/logutil"`

Reply:
yes, we should use the full path